### DISCUSSION TITLE
Help me enable running scripts with ublock origin

### DISCUSSION THREAD

**Discussion by hankypoo7** [3kh0/ext-remover#860] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i have ublock origin and followed the instructions on [this page](https://github.com/3kh0/ext-remover?tab=readme-ov-file#ublock-run-run-code-on-pages), but it won't work. what did i do wrong? this is my settings for ublock origin:
![Screenshot 2023-12-14 2 23 19 PM](https://github.com/3kh0/ext-remover/assets/100436822/44b9ee28-55ca-4db3-bf18-d66744a441e1)
![Screenshot 2023-12-14 2 23 07 PM](https://github.com/3kh0/ext-remover/assets/100436822/bd9c49f1-8b72-44e8-babc-aee377508312)
also, when i get prompted to "eval:" by the popup i have no idea what that means. Am i just supposed to put in my javascript code?

**Comment by funkymokey** [comment, marked as answer]
just use skiovox lol

**Reply by hankypoo7** [reply]
holy crap it worked
